subsys/net/ip/net_stats.c

```diff
@@ -4,14 +4,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define LOG_MODULE_NAME net_stats
-
 #if defined(CONFIG_NET_STATISTICS_PERIODIC_OUTPUT)
 #define NET_LOG_LEVEL LOG_LEVEL_INF
```

Comment:
imo, this could also be resolved to log level otherwise it overwrites kconfig setting which may be confusing.

Reply:
yes, i was actually questioning presence of `CONFIG_NET_STATISTICS_PERIODIC_OUTPUT`. If we have logging level setting that might be enough. I was considering situation when someone (in the future when different log levels are present in that file) sets logs to DBG enables periodic output and wonders why DBG output is not seen.